docs/source/configuration.rst

```diff
@@ -644,6 +644,7 @@ You might have arguments that you pass through every time, e.g rules you
     [sqlfluff]
     verbose = 1
     exclude_rules = L022,L027
+    fix_even_unparsable = True
```

Comment:
I was unsure. Having it here makes people aware that it can be configured, but if we don't explain that it's a bad idea, it may encourage people to use it inappropriately. I'll remove it.